### DISCUSSION TITLE
❗️❗️2.x 会做的一些事

### DISCUSSION THREAD

**Discussion by anlyyao** [Tencent/tdesign-mobile-vue#1146] [Show and tell]
## DropdownMenu
-[ 该组件 CSS Variables 命名](https://github.com/Tencent/tdesign-common/blob/458fb5ce95e8f2529297302dcb3b27c8f0aa5c3f/style/mobile/components/dropdown-menu/v2/_var.less)存在错误，如 `--td-dropdown-menu-font-sizem` 应为 `--td-dropdown-menu-font-size`

**Comment by liweijie0812** [comment]
List组件: `scroll` 事件参数 `(bottomDistance: number, scrollTop: number)` 改 `(options: { e: Event | WheelEvent; scrollTop: number; scrollBottom: number })`

**Comment by liweijie0812** [comment]
Form:  样式类名 `t-form__item`  改 `t-form-item` https://github.com/Tencent/tdesign-common/pull/1818

**Comment by Jesse205** [comment]
您好，我发现现在1.x的common没有开启ts的严格模式，且本组件库和common都没有开启空安全检查，这是为什么呢？2.x有计划开启吗？
